tests/unit/s2n_change_cipher_spec_test.c

```diff
@@ -53,7 +53,14 @@ int main(int argc, char **argv)
         EXPECT_SUCCESS(s2n_connection_free(conn));
     }
 
-    /* Test that s2n_basic_ccs_recv errors on wrong change cipher spec types */
+    /* Test that s2n_basic_ccs_recv errors on wrong change cipher spec types
+    *= https://tools.ietf.org/rfc/rfc8446#5
+    *= type=test
+    *# An
+    *# implementation which receives any other change_cipher_spec value or
```

Comment:
Nit: I think newer duvet can handle whitespace changes? You might not need "An" alone on one line.